tests/modules/nf-core/chewbbaca/createschema/test.yml

```diff
@@ -0,0 +1,234 @@
+- name: chewbbaca createschema test_chewbbaca_createschema
+  command: nextflow run ./tests/modules/nf-core/chewbbaca/createschema -entry test_chewbbaca_createschema -c ./tests/config/nextflow.config
+  tags:
+    - chewbbaca
+    - chewbbaca/createschema
+  files:
+    - path: output/chewbbaca/results/cds_coordinates.tsv
+      md5sum: 7fef49207aa024b68f465e986f4c7444
+    - path: output/chewbbaca/results/invalid_cds.txt
+      md5sum: d0afbd9996054d8838e62dc3e852fa07
+    - path: output/chewbbaca/results/test/.genes_list
+      contains:
+        - "# TODO nf-core: file md5sum was variable, please replace this text with a string found in the file instead "
```

Comment:
There are some TODOs here in the text. Can you substitute the TODO with any string that can be found inside the file? Also for the other occurrences. Maybe this will make the tests pass.

Reply:
Can't figure out how to fix this issue. 

the output of the file .genes_list looks like this 

```
00000000  80 04 95 82 01 00 00 00  00 00 00 5d 94 28 8c 17  |...........].(..|
00000010  63 6f 6e 74 69 67 73 2d  70 72 6f 74 65 69 6e 31  |contigs-protein1|
00000020  39 2e 66 61 73 74 61 94  8c 17 63 6f 6e 74 69 67  |9.fasta...contig|
00000030  73 2d 70 72 6f 74 65 69  6e 31 30 2e 66 61 73 74  |s-protein10.fast|
00000040  61 94 8c 17 63 6f 6e 74  69 67 73 2d 70 72 6f 74  |a...contigs-prot|
00000050  65 69 6e 32 30 2e 66 61  73 74 61 94 8c 16 63 6f  |ein20.fasta...co|
00000060  6e 74 69 67 73 2d 70 72  6f 74 65 69 6e 36 2e 66  |ntigs-protein6.f|
00000070  61 73 74 61 94 8c 17 63  6f 6e 74 69 67 73 2d 70  |asta...contigs-p|
00000080  72 6f 74 65 69 6e 31 38  2e 66 61 73 74 61 94 8c  |rotein18.fasta..|
00000090  16 63 6f 6e 74 69 67 73  2d 70 72 6f 74 65 69 6e  |.contigs-protein|
000000a0  38 2e 66 61 73 74 61 94  8c 15 67 65 6e 6f 6d 65  |8.fasta...genome|
000000b0  2d 70 72 6f 74 65 69 6e  31 2e 66 61 73 74 61 94  |-protein1.fasta.|
000000c0  8c 17 63 6f 6e 74 69 67  73 2d 70 72 6f 74 65 69  |..contigs-protei|
000000d0  6e 31 36 2e 66 61 73 74  61 94 8c 16 63 6f 6e 74  |n16.fasta...cont|
000000e0  69 67 73 2d 70 72 6f 74  65 69 6e 31 2e 66 61 73  |igs-protein1.fas|
000000f0  74 61 94 8c 16 63 6f 6e  74 69 67 73 2d 70 72 6f  |ta...contigs-pro|
00000100  74 65 69 6e 37 2e 66 61  73 74 61 94 8c 17 63 6f  |tein7.fasta...co|
00000110  6e 74 69 67 73 2d 70 72  6f 74 65 69 6e 31 34 2e  |ntigs-protein14.|
00000120  66 61 73 74 61 94 8c 16  63 6f 6e 74 69 67 73 2d  |fasta...contigs-|
00000130  70 72 6f 74 65 69 6e 34  2e 66 61 73 74 61 94 8c  |protein4.fasta..|
```

Seems like this file doesn't have a string output. any suggestions? @ggabernet